roles/network_plugin/cilium/tasks/install.yml

```diff
@@ -91,7 +91,7 @@
 
 - name: Cilium | Copy Ciliumcli binary from download dir
   copy:
-    src: "{{ downloads.ciliumcli.dest }}"
+    src: "{{ local_release_dir }}/cilium"
```

Comment:
HI @tu1h 

Does the `downloads.ciliumcli.dest` not work correctly ？

Other binary usually use the {{ downloads.XX.dest }} like : https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/skopeo/tasks/main.yml#L30

Reply:
https://github.com/kubernetes-sigs/kubespray/blob/267a8c6025e6b3e61e8ff516e7659a38af2b68a5/roles/download/defaults/main/main.yml#L163

https://github.com/kubernetes-sigs/kubespray/blob/267a8c6025e6b3e61e8ff516e7659a38af2b68a5/roles/download/defaults/main/main.yml#L178

The downloading for cilium cli is the `.tar.gz` which is different from skopeo as a static executable `ELF`. So they should not use the same process way.

Reply:
> https://github.com/kubernetes-sigs/kubespray/blob/267a8c6025e6b3e61e8ff516e7659a38af2b68a5/roles/download/defaults/main/main.yml#L163
> 
> https://github.com/kubernetes-sigs/kubespray/blob/267a8c6025e6b3e61e8ff516e7659a38af2b68a5/roles/download/defaults/main/main.yml#L178
> 
> The downloading for cilium cli is the `.tar.gz` which is different from skopeo as a static executable `ELF`.
> 
> the `downloads.ciliumcli.dest` is still a `.tar.gz` file, that's describe at the beginning of the PR.

I understand, the extraction of the tar-ball isn't referenced by ```downloads.ciliumcli.dest```. So, it is necessary to point to the cilium cli binary directly.